apps/nextra/pages/en/build/sdks/community-sdks.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: "Community SDKs"
+---
+
+# Community SDKs
+
+Here is a list of community built SDKs for Aptos.  These may not be fully vetted by the Aptos team, and may still be in development.
+They are still provided as a resource for all developers.
+
+## SDKs
+
+- [Swift SDK](./community-sdks/swift-sdk)
+
+{/* TODO Move Kotlin SDK here */}
```

Comment:
Should we include the Kotlin SDK move in this PR or do you wanna do it in a follow up?

Reply:
I want to follow up after the Kotlin SDK PR is closed.  No reason to make him refactor his PR for now.